README.md

```diff
@@ -1,40 +1,7 @@
-<a name="readme-top"></a>
 
-<!--
-HOW TO USE:
-This is an example of how you may give instructions on setting up your project locally.
-
-Modify this file to match your project and remove sections that don't apply.
```

Comment:
Great job using the inline comments for giving the review and highlighting the correct part of the project. However I would suggest a few changes to make your communication better.

- [ ] Please make sure that you phrase the comments politely, as suggestions instead of orders. 

For example:

*  This is a polite suggestion: "I think it would be better to remove all commented out lines in README.md"
*  This reads as an order: "You need to remove all instructions in README.md"

- [ ] Remember to always include an explanation, a "why", for each requested change - it might be obvious for you but not for someone else . For example:

* This is a well-explained request: *Why don't you use an article tag for the .first-article element instead of a div, that would improve the HTML structure and make the page more accessible*
* This request lacks a "why": *Why don't you use an article tag here instead of a div?*

Here is a link to [the part of the lesson about communication](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/code-review/articles/give_code_review_basics.md#how-to-communicate) that you can use 😄.